packages/asc-ui/src/components/Heading/Heading.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { styledComponents } from '@datapunt/asc-core'
+import HeadingStyle from './HeadingStyle'
+
+const Heading: React.FC<
+  styledComponents.StyledProps<any> & React.HTMLAttributes<HTMLHeadingElement>
```

Comment:
Replace HTMLAttributes type, which is huge with a small custom type/interface containing just the className